pinia-orm/axios v2.0.0-alpha.1 still looking for pinia-orm >=1.9.1 peer dependency

### Environment

Vue 3, Node 22.13.1

### Reproduction

- `npm install @pinia-orm v2.0.0-alpha.1`
- `npm install @pinia-orm/axios@2.0.0-alpha.1`

### Describe the bug

Since I'm on Pinia v3 and already had Pinia-ORM, I was trying to setup the axios extension, which didn't seem able to resolve (see logs).
I've used --force for now, which works.

-> https://github.com/CodeDredd/pinia-orm/blob/95b1d0ae3dd43922c713b92d2813703c1e6bacaa/packages/axios/package.json#L57

### Logs

```shell-script
npm i @pinia-orm/axios@2.0.0-alpha.1 --save
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: [redacted-project-name]@2.0.0
npm error Found: pinia-orm@2.0.0-alpha.1
npm error node_modules/pinia-orm
npm error   pinia-orm@"^2.0.0-alpha.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer pinia-orm@">=1.9.1" from @pinia-orm/axios@2.0.0-alpha.1
npm error node_modules/@pinia-orm/axios
npm error   @pinia-orm/axios@"2.0.0-alpha.1" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/packages/axios/package.json
+++ b/packages/axios/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@pinia-orm/axios",
-  "version": "2.0.0-alpha.1",
+  "version": "2.0.0-alpha.2",
   "description": "Axios plugin for pinia-orm",
   "bugs": {
     "url": "https://github.com/CodeDredd/pinia-orm/issues"

--- a/packages/normalizr/package.json
+++ b/packages/normalizr/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@pinia-orm/normalizr",
-  "version": "2.0.0-alpha.1",
+  "version": "2.0.0-alpha.2",
   "description": "Normalizes JSON according to schema for Redux and Flux applications",
   "bugs": {
     "url": "https://github.com/CodeDredd/pinia-orm/issues"

--- a/packages/nuxt/package.json
+++ b/packages/nuxt/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@pinia-orm/nuxt",
   "description": "Nuxt module for pinia-orm",
-  "version": "2.0.0-alpha.1",
+  "version": "2.0.0-alpha.2",
   "license": "MIT",
   "author": {
     "name": "Gregor Becker",

--- a/packages/pinia-orm/package.json
+++ b/packages/pinia-orm/package.json
@@ -1,6 +1,6 @@
 {
   "name": "pinia-orm",
-  "version": "2.0.0-alpha.1",
+  "version": "2.0.0-alpha.2",
   "description": "The Pinia plugin to enable Object-Relational Mapping access to the Pinia Store.",
   "keywords": [
     "vue",

